director/sort.go

```diff
@@ -286,7 +296,19 @@ func sortServerAds(ctx context.Context, clientAddr netip.Addr, ads []server_stru
 	weights := make(SwapMaps, len(ads))
 	sortMethod := param.Director_CacheSortMethod.GetString()
 	// This will handle the case where the client address is invalid or the lat/long is not resolvable.
-	clientCoord := getClientLatLong(ctx, clientAddr)
+	clientCoord, err := getClientLatLong(clientAddr)
+	if err != nil {
```

Comment:
Same comment about this section -- error should be printed regardless of error type